doc/source/cluster/index.rst

```diff
@@ -4,42 +4,69 @@
 
 .. _cluster-index:
 
-Ray Cluster Overview
-====================
+Ray Clusters Overview
+=====================
 
 What is a Ray cluster?
-------------------------
+----------------------
 
-One of Ray's strengths is the ability to leverage multiple machines in the same
-program. Ray can, of course, be run on a single machine (and is done so often),
-but the real power is using Ray on a cluster of machines.
+One of Ray's strengths is the ability to leverage multiple machines for distributed execution. Ray can, of course, be run on a single machine (and is done so often), but the real power is using Ray on a cluster of machines.
```

Comment:
I prefer shorter lines for readability on the part of authors (not everyone has a good soft-wrap setup), but I'm not adamant about this. I wouldn't mind to roughly respect the 88 chars our linter imposes on other projects.